src/util/index.js

```diff
@@ -35,3 +35,11 @@ export const dispatchEvent = (el, etype) => {
   evObj.initEvent(etype, true, false)
   el.dispatchEvent(evObj)
 }
+
+export const coinToss = () =>
+  Math.floor(Math.random() * 2) === 0
+
+export const flowLog = message => d => {
```

Comment:
looks like this isn't used? if you want to keep it in for debugging purposes, let's call it tapLog instead